### DISCUSSION TITLE
Simplicity and speed over security

### DISCUSSION THREAD

**Discussion by jamesliley** [community/community#45417] [Code Security]
### Select Topic Area

Product Feedback

### Body

I would rather not need to verify a new device by a code sent to e-mail. I also don't want to have to use access tokens. I definitely don't want 2FA.

These things may only take five minutes but they take five minutes over and over again. 

I really don't care about the security of my own repositories. Nothing in them can be used for ill, and I don't care if they are plagiarised. 

Surely a lot of repositories are like this. By all means insist on security features if I'm wanting to access other people's shared repositories, but why do I need unwanted security features in order to access my own?

I can't seem to find a way to turn all of these security features off. Is there one?

Thanks for any help.

**Comment by jge162** [comment]
Hi @jamesliley To disable 2FA, you can go to your GitHub account settings and select "Security" from the left sidebar. Under "Two-Factor Authentication," click "Edit" and then "Deactivate Two-Factor Authentication."

2FA, email verification, and access tokens are in place to protect your account and your repositories from unauthorized access. While you may not consider your personal repositories to contain sensitive information, it's still important to maintain proper security practices to protect your account and your work.

hope this helps. Bests to you.

**Reply by jge162** [reply]
I understand your frustration.

**Reply by jamesliley** [reply]
Hi. Thanks for the response. This is helpful, though I would appreciate the option to not use email verification or access tokens. Perhaps this could be considered in future updates. Best, James.

**Reply by airtower-luna** [reply]
GitHub has announced they'll require 2FA for anyone who contributes code by the end of this year: https://github.blog/2022-05-04-software-security-starts-with-the-developer-securing-developer-accounts-with-2fa/ So I'm pretty sure they won't make a move in the opposite direction. :sweat_smile:

**Comment by DNin01** [comment]
GitHub has started gradually rolling out a requirement to enable 2FA as of earlier today (or yesterday depending on your time zone), but on supported devices including Windows, Android and iOS devices, you can also use your computer or phone as a security key, which is called a passkey. It's super easy to use and fast - it's basically like unlocking your device to sign in, so you don't really lose any time. Hopefully that satisfies your request, maybe along with SMS as a fallback option.

Was that your question, by the way?
